src/auth/auth-get-domain.ts

```diff
@@ -0,0 +1,33 @@
+const thirdPartyDomainSTSUrlMappings: { [domain: string]: string; } = {
+  'bbk12.com': 'https://sts-sso.bbk12.com'
+};
+const defaultSTSUrl = 'https://s21aidntoken00blkbapp01.nxt.blackbaud.com';
+
+export class BBAuthGetDomain {
+
+  private static cacheDictionary: any = {};
+
+  public static isRegisteredDomain(domain: string = window.location.hostname): boolean {
```

Comment:
Change this to `getRegisteredDomain(): string` and return the domain rather than a flag. This can then be used in `getSTSDomain()` to grab the STS URL and eliminate the need for the `cacheDictionary` property.